libraries/anvill_passes/tests/src/RecoverStackFrameInformation.cpp

```diff
@@ -35,11 +35,16 @@ namespace anvill {
 
 TEST_SUITE("RecoverStackFrameInformation") {
   TEST_CASE("Run the whole pass on aaa well-formed function") {
-    static const std::array<bool, 2> kZeroInitStackSettings = {false, true};
+    static const std::vector<StackFrameStructureInitializationProcedure>
```

Comment:
You can also do: `static const auto kInitStackSettings[] = { ... };` and this will infer the enumeration type, and work in range-based for loops :-D